src/views/DataExploration/components/PageContent/index.tsx

```diff
@@ -63,6 +64,15 @@ import styles from './index.module.scss';
 const { Title } = Typography;
 
 export const MAX_TITLE_LENGTH = 200;
+
+export enum FilterActionType {
+  UPDATE_FILER = 'UPDATE_FILER',
```

Comment:
Ça ne serait pas filter ?
```suggestion
  UPDATE_FILTER = 'UPDATE_FILTER',
```